package.json

```
@@ -15,6 +15,7 @@
     "build:bundle": "lerna run build:bundle",
     "build:watch": "lerna exec --parallel -- 'yarn run build:watch'",
     "build:ifchanged": "lerna exec -- ../../scripts/build_if_changed.sh",
+    "check-specs": "./scripts/check-specs.sh",
```

```suggestion
    "check-specs": "sh ./scripts/check-specs.sh",
```
probably do this to avoid relying on file permission